jlm/rvsdg/region.cpp

```diff
@@ -515,6 +515,39 @@ Region::ToString(const util::Annotation & annotation, char labelValueSeparator)
   return util::strfmt(annotation.Label(), labelValueSeparator, value);
 }
 
+static size_t
+computeDepth(const Node & node, std::unordered_map<const Node *, size_t> & depthMap)
+{
+  if (depthMap.find(&node) != depthMap.end())
```

Comment:
more canonical would be to use:

auto i = depthMap.find(&node);
if (i != depthMap.end()) { return i->second; }

to avoid double search

Reply:
Fixed